etc/releasing.md

```diff
@@ -331,6 +331,16 @@ Commit the updates to `CHANGELOG.md`.
 git commit -m 'Update CHANGELOG for X.Y.Z'
 ```
 
+### Update GitHub Webhook
+
+For a non-patch release, update the [Github Webhook](https://wiki.corp.mongodb.com/display/INTX/Githook) to include the new branch.
+
+Navigate to the [Webhook Settings](https://github.com/mongodb/mongo-cxx-driver/settings/hooks)
+
+Click `Edit` on the hook for `https://githook.mongodb.com/`.
+
+Add the branch to the `Payload URL`.
```

Comment:
Recommend keeping only actively-maintained branches in the list, which I believe would currently mean the following:

```
https://githook.mongodb.com/?branches=master..releases/v3.11..releases/v4.0
```